Common.mk

```diff
@@ -688,7 +688,9 @@ release: $(RELEASE_TARGETS)
 .PHONY: %/release-branches/all
 %/release-branches/all:
 	@for version in $(SUPPORTED_K8S_VERSIONS) ; do \
-		$(MAKE) $* RELEASE_BRANCH=$$version; \
+	    if [ -d "$$version" ]; then \
```

Comment:
Let's just add a comment, above the target, saying something about how we allow projects to skip building specific versions by not including those folders. It's admittedly a bit magical and potentially will hide issues if someone accidentally forgets to add all the folders, I just want this documented in some way for future us to try and determine what the hell we were thinking.